src/verify.py

```diff
@@ -0,0 +1,6 @@
+def verify(l:list[int], v:int ) -> bool:
```

Comment:
Sarebbe meglio usare dei nomi più descrittivi sia per il nome della funzione (es: is_element_in_list oppure list_contains), sia per i parametri in ingresso(che chiamerei senza abbreviazione list e value), in modo che, semplicemente leggendo la firma della funziona, si intuisca cosa si aspetta in input, cosa ci aspettiamo che faccia e cosa eventualmente ritorna come output. 
In questo caso potrebbe sembrare superfluo perché la funzione è banale, tuttavia è una buona abitudine che è meglio acquisire il prima possibile :)